frontend/app/component/pollingstation/PollingStationFormNavigation.tsx

```diff
@@ -152,7 +152,7 @@ export function PollingStationFormNavigation({ pollingStationId, election }: Pol
         </Modal>
       )}
 
-      {error && <Feedback id="feedback-server-error" type="error" data={error} />}
+      {apiError && <Feedback id="feedback-server-error" type="error" apiError={apiError} />}
```

Comment:
Once this kind of error is shown, it's shown on all pages. It also does not go away after a succesful submit.

Tested by making the finalise endpoint return a 500 response.